api-go/config/multi-repo.yaml

```diff
@@ -41,7 +42,6 @@ apache/skywalking:
     - apache/skywalking-infra-e2e
     - apache/skywalking-showcase
     - apache/skywalking-docker
-    - apache/skywalking-booster-ui
```

Comment:
We could keep this, I will add this to skywalking side soon.

Reply:
OK